sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/Utility.java

```diff
@@ -871,6 +903,40 @@ private static AssessmentSentiment toAssessmentSentiment(SentenceAssessment sent
         return assessmentSentiment;
     }
 
+    private static ExtractSummaryResult toExtractSummaryResult(ExtractedDocumentSummary documentSummary) {
+        final List<ExtractedSummarySentence> sentences = documentSummary.getSentences();
+        final List<SummarySentence> summarySentences = sentences.stream().map(sentence -> {
+            final SummarySentence summarySentence = new SummarySentence();
+            SummarySentencePropertiesHelper.setText(summarySentence, sentence.getText());
+            SummarySentencePropertiesHelper.setRankScore(summarySentence, sentence.getRankScore());
+            SummarySentencePropertiesHelper.setLength(summarySentence, sentence.getLength());
+            SummarySentencePropertiesHelper.setOffset(summarySentence, sentence.getOffset());
+            return summarySentence;
+        }).collect(Collectors.toList());
+
+        // Warnings
+        final List<TextAnalyticsWarning> warnings = documentSummary.getWarnings().stream().map(
+            warning -> {
+                final WarningCodeValue warningCodeValue = warning.getCode();
+                return new TextAnalyticsWarning(
+                    WarningCode.fromString(warningCodeValue == null ? null : warningCodeValue.toString()),
+                    warning.getMessage());
+            }).collect(Collectors.toList());
```

Comment:
This seems like repeated code, is there away to reuse?

Reply:
Yes. It can be a helper method